### ISSUE TITLE
Date

### ISSUE THREAD

**Issue by vedantvohra** [lmeninato/GoFundMe#3]
Date is a crucial feature missing from the data

**Comment by lmeninato** [comment]
Yeah decent point. I think August 15, 2018 is the appropriate date for the dataset (based on this [commit](https://github.com/lmeninato/GoFundMe/commit/dfb36ee44592a912b214a2011619b786487d69a4)).
Feel free to make a pull request adding that date column to the csv file. That is the date I compiled the list of urls to scrape.
